libs/domains/services/feature/src/lib/service-links-dropdown/service-links-dropdown.tsx

```diff
@@ -0,0 +1,69 @@
+import { type Link } from 'qovery-typescript-axios'
+import {
+  ButtonLegacy,
+  ButtonLegacySize,
+  ButtonLegacyStyle,
+  CopyToClipboard,
+  IconAwesomeEnum,
+  Popover,
+  Skeleton,
+  Truncate,
+} from '@qovery/shared/ui'
+import { plural } from '@qovery/shared/util-js'
+
+export interface ServiceLinksDropdownProps {
+  links?: Link[]
+}
+
+// TODO: Update props and using React Query
+export function ServiceLinksDropdown({ links }: ServiceLinksDropdownProps) {
```

Comment:
Not sure about the naming here.
IMO it should be named after the functional behavior. (ServicePreviewLinks ?)
And if not, it should be `ServiceLinksPopover`.
We can talk about it.

Reply:
`ServiceLinksPopover` looks better yes, and it's a Popover, not a dropdown as well

Reply:
https://github.com/Qovery/console/pull/859/commits/3fd725f5aa63a38744861a8930a1106b9ba56f74